app/views/admin/teachers/_teacher.html.erb

```diff
@@ -0,0 +1,9 @@
+  <tr>
+    <td><%= teacher.user.full_name %></td>
```

Comment:
delegate these user's properties to teacher

Reply:
delegated